src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -175,7 +183,8 @@ public static void GetFileNameWithoutExtension(string path, string expected)
         public static void GetPathRoot()
         {
             Assert.Null(Path.GetPathRoot(null));
-            Assert.Equal(string.Empty, Path.GetPathRoot(string.Empty));
+
+            AssertExtensions.Throws<ArgumentException>("path", null, () => Path.GetPathRoot(string.Empty));
```

Comment:
This was also fixed here:
https://github.com/dotnet/corefx/pull/19440/commits/e1755b5a62cd07037d5488d459a61cf97378226d#diff-19631815f66f1f0ad0492da989a7463cR186